templates/vue/src/components/SettingsModal.vue

```diff
@@ -1,83 +1,105 @@
 <template>
-  <b-modal id="settings-modal" size="lg" title="Tapestry Settings" body-class="p-0">
+  <b-modal
+    id="settings-modal"
+    size="lg"
+    title="Tapestry Settings"
+    body-class="p-0"
+  >
     <b-container fluid class="px-0">
       <b-tabs card>
         <b-tab title="Appearance" active>
-          <b-form-group label="Background URL" description="Add a background image to the page where this tapestry 
-              appears.">
+          <b-form-group
+            label="Background URL"
+            description="Add a background image to the page where this tapestry
+              appears."
+          >
             <b-form-input
               id="background-url"
-              placeholder="Enter background URL"
               v-model="backgroundUrl"
+              placeholder="Enter background URL"
               autofocus
               required
             />
           </b-form-group>
-          <b-form-group label="Auto-Layout" description="With auto-layout enabled, nodes will place themselves in
+          <b-form-group
+            label="Auto-Layout"
+            description="With auto-layout enabled, nodes will place themselves in
               the best position possible. If this is disabled, you will need to manually place the nodes where
-              you want them to appear.">
-            <b-form-checkbox v-model="autoLayout" switch>{{ autoLayout ? 'Enabled' : 'Disabled' }}</b-form-checkbox>
+              you want them to appear."
+          >
+            <b-form-checkbox v-model="autoLayout" switch>{{
+              autoLayout ? "Enabled" : "Disabled"
+            }}</b-form-checkbox>
           </b-form-group>
         </b-tab>
       </b-tabs>
     </b-container>
     <template slot="modal-footer">
-      <p class="mb-0 p-0 text-muted small"><strong>Note:</strong> Page will refresh when you save to apply your new settings.</p>
+      <p class="mb-0 p-0 text-muted small">
+        <strong>Note:</strong> Page will refresh when you save to apply your new
+        settings.
+      </p>
       <span style="flex-grow:1;"></span>
-      <b-button size="sm" variant="secondary" @click="closeModal">Cancel</b-button>
-      <b-button size="sm" variant="primary" @click="updateSettings">Save</b-button>
+      <b-button size="sm" variant="secondary" @click="closeModal"
+        >Cancel</b-button
+      >
+      <b-button size="sm" variant="primary" @click="updateSettings"
+        >Save</b-button
+      >
     </template>
   </b-modal>
 </template>
 
 <script>
 export default {
-  name: 'settings-modal',
-  data() {
-    return {
-      currentSettings: {},
-      backgroundUrl: '',
-      autoLayout: false,
-    }
-  },
+  name: "settings-modal",
```

Comment:
How come this was changed to camelCase in the other modal, but not here? I thought it was a lint setting.. Let's keep it consistent.